### PR TITLE
python-utils-r1.eclass: disable py3.8

### PR DESCRIPTION
**Pull request by SoapGentoo**
Signed-off-by: David Seifert <soap@gentoo.org>

### CODE REVIEW

**Comment by SoapGentoo** [comment]
@gentoo/python

File: eclass/python-utils-r1.eclass [hunk 1]
```diff
@@ -129,9 +129,9 @@ _python_set_impls() {
 			# please keep them in sync with _PYTHON_ALL_IMPLS
 			# and _PYTHON_HISTORICAL_IMPLS
```
Comment (mgorny) [inline eclass/python-utils-r1.eclass:130]: You didn't read the comment, did you?

Comment (SoapGentoo) [reply eclass/python-utils-r1.eclass:130]: yup sorry, updated now

**Comment by arthurzam** [comment]
ACK from me for disabling py3.8 - we have listened for the request to postpone it till 2023-01.
As also good news, we are nearly ready with migration to py3.10, meaning when we desire (based on plan, ML, etc) we would be able drop py3.9.

File: dev-python/time-machine/time-machine-2.9.0.ebuild [hunk 2]
```diff
@@ -25,12 +25,5 @@ KEYWORDS="~amd64 ~ppc ~riscv"
 RDEPEND="
 	dev-python/python-dateutil[${PYTHON_USEDEP}]
 "
-BDEPEND="
```
Comment (mgorny) [inline dev-python/time-machine/time-machine-2.9.0.ebuild:28]: I get that it doesn't really change anything but perhaps remove 3.8 from PYTHON_COMPAT to make it more correct.

**Review by mgorny — COMMENTED** [review]
There's at least `profiles/desc` to update as well, `python3_8` masks/forces in `profiles`.

I suppose there's no doc for this yet because I was supposed to write one when we remove the next impl. From https://projects.gentoo.org/python/guide/interpreter-maintenance.html#adding-a-new-python-implementation, the following seems to apply:

- `PYTHON_PKG_DEP` removal
- removing matching eclass tests
- gpyutils list update

File: profiles/desc/python_targets.desc [hunk 3]
```diff
@@ -1,9 +1,8 @@
-# Copyright 1999-2021 Gentoo Authors.
+# Copyright 1999-2023 Gentoo Authors.
 # Distributed under the terms of the GNU General Public License v2
 
 # This file contains descriptions of PYTHON_TARGETS USE_EXPAND flags.
 
-python3_8 - Build with Python 3.8
```
Comment (mgorny) [inline profiles/desc/python_targets.desc:6]: single_targets too.

**Pull request closed without merging**